frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-page-content/feature-flag-root-section-card/feature-flag-root-section-card.component.ts

```diff
@@ -88,6 +91,10 @@ export class FeatureFlagRootSectionCardComponent {
     );
   }
 
+  getAllFeatureFlagIds() {
```

Comment:
There's no point in passing this data in from the component, use this selector in the effect. The component doesn't need to be involved in getting the ids to pass, this kind of thing just creates more code to manage local state, as I've put in other comments.

Reply:
this can just be removed now, we don't need multiple designs exported